lgc/patch/FragColorExport.cpp

```diff
@@ -630,8 +632,13 @@ CallInst *LowerFragColorExport::addExportForGenericOutputs(Function *fragEntryPo
     }
   }
 
-  if (colorExports.empty())
+  if (colorExports.empty()) {
+    SmallVector<ColorExportInfo, 8> info;
+    const auto &builtInUsage = m_resUsage->builtInUsage.fs;
+    bool hasDepthExpFmtZero = !(builtInUsage.sampleMask || builtInUsage.fragStencilRef || builtInUsage.fragDepth);
+    m_pipelineState->getPalMetadata()->updateSpiShaderColFormat(info, hasDepthExpFmtZero, builtInUsage.discard);
```

Comment:
I thought I had fixed this one.  I must have lost it in when merging or something.  Thanks for fixing this.